includes/payment-methods/class-digital-wallets-payment-gateway.php

```diff
@@ -44,19 +43,4 @@ public function __construct( WC_Payments_API_Client $payments_api_client, WC_Pay
 		$this->title              = __( 'Digital Wallets', 'woocommerce-payments' );
 		$this->description        = __( 'Mandate Information.', 'woocommerce-payments' );
 	}
-
-	/**
```

Comment:
No longer needed

Reply:
Those are the default values that WCPay sets when no options are checked: https://github.com/Automattic/woocommerce-payments/blob/b7416935a233242c959fdd93a169b094b1e16eaa/includes/class-wc-payment-gateway-wcpay.php#L251-L254

The same is happening on the `develop` branch with the "Grouped settings" flag unchecked (i.e.: with the "older" settings page).

I have been thinking whether or not we should be changing them.
If you'd like to, I can do this: 794681dd